### PR TITLE
Add test for multiple endpoints with the same IP.

### PR DESCRIPTION
**Pull request by alexaltair**

### CODE REVIEW

File: calico_containers/tests/st/test_duplicate_ips.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        Start two workloads with the same IP on different hosts. Make sure they
+        can be reached from all places even after one of them is deleted.
+        """
+        host1 = DockerHost('host1')
+        host2 = DockerHost('host2')
+        host3 = DockerHost('host3')
+
+        calicoctl = "/code/dist/calicoctl %s"
+
+        host1.execute(calicoctl % ("node --ip=%s" % host1.ip))
```
Comment (fasaxc) [inline calico_containers/tests/st/test_duplicate_ips.py]: How about adding a `calicoctl()` method to the host?  Then you could do something like `host1.calicoctl("node", ip=host1.ip)` and get rid of all those string interpolations.

File: calico_containers/tests/st/test_duplicate_ips.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        host2.execute(calicoctl % "profile TEST_PROFILE member add workload2")
+        host2.execute(calicoctl % "profile TEST_PROFILE member add dup2")
+        host3.execute(calicoctl % "profile TEST_PROFILE member add workload3")
+
+        # Wait for the workload networking to converge.
+        ping = partial(host1.execute, "docker exec workload1 ping -c 4 192.168.1.4")
+        self.retry_until_success(ping, ex_class=ErrorReturnCode_1)
+
+        # Check for standard connectivity
+        host1.execute("docker exec workload1 ping -c 4 192.168.1.4")
```
Comment (fasaxc) [inline calico_containers/tests/st/test_duplicate_ips.py]: Likewise, this could be simplified a lot by having `host1.assert_can_ping(...)`

**Comment by fasaxc** [comment]
Looks pretty straightforward, which is a good sign.  I do think it's worth cleaning up the duplication at this stage.  I expect it'll pay back on the next test.

Seems like there could be some other useful tests that start with this topology.  Make sure you keep the topology setup common as you add more.

**Comment by alexaltair** [comment]
My next branch is cleaning up the things you mentioned et al. Is this one good to go?

**Pull request merged**